### ISSUE TITLE
The issue with closing the side panel

### ISSUE THREAD

**Issue by n4ze3m** [n4ze3m/page-assist#40]
What I know is that the side panel in Chromium-based browsers is messed up:

- Chrome and Edge only have a stable side panel as far as I've tested.
- Brave doesn't have a side panel close button: [Link to Brave Community](https://community.brave.com/t/chromium-sidepanel-is-broken-doesnt-show-close-btn-neither-shows-extension-icon/511146).
- Arc and Opera do not support side panels yet.

**Comment by yihou** [comment]
how about implement like how Monica does, able to open in browser's view and have options to enable for Chromium side panel view or in browser

**Comment by n4ze3m** [comment]
Yes, this is a great idea. I will look into it, even though the 0.0.1 version of Page Assist is like that 😅 

![image](https://github.com/n4ze3m/page-assist/assets/39720973/12c79d60-3f3e-473c-91b2-89aabbb8e35b)
